src/main/java/com/iexec/core/task/TaskService.java

```diff
@@ -53,26 +45,16 @@
     private final TaskUpdateRequestManager taskUpdateRequestManager;
     private final IexecHubService iexecHubService;
     private final ReplicatesService replicatesService;
-    private final ApplicationEventPublisher applicationEventPublisher;
-    private final WorkerService workerService;
-    private final BlockchainAdapterService blockchainAdapterService;
 
     public TaskService(
             TaskRepository taskRepository,
             TaskUpdateRequestManager taskUpdateRequestManager,
```

Comment:
Do we have taskUpdateRequestManager usages here?

Reply:
We don't, thanks for the catch!